CDCgov/prime-reportstream
# 7048 Universal Pipeline - Skip Inactive Receivers

This PR adds an early check for a potential receiver's status, and skips those that are set to `INACTIVE`

Test Steps:
1. Run automated tests, specifically `FhirRouterTests`.
2. To test manually, submit a report. After receiving a response for that submission including the `destinations` array, go to `public.setting` on the database and set these destinations to `is_active=false`.
3. Submit the same report again and verify it's not being sent to the inactive destinations.

## Changes
- Change the `CustomerStatus` check during the `addReceivers` step to skip the loop if it's set to `INACTIVE`
- Add test case and data to cover the new logic

## Linked Issues
- #7048
